components/builder-api-client/src/builder.rs

```diff
@@ -57,6 +58,9 @@ const X_FILENAME: &str = "x-filename";
 
 const DEFAULT_API_PATH: &str = "/v1";
 
+pub const API_RETRIES: usize = 5;
+pub const API_RETRY_WAIT: Duration = Duration::from_millis(10000);
```

Comment:
Unless it's really important to use the scale of milliseconds here, this would be more readable as `Duration::from_secs(10)`.

PLUS YOU GET TO SAVE THREE ZEROES!!!! :tada:

Reply:
Added in https://github.com/habitat-sh/habitat/pull/7874/commits/2a457e8b6f09ace44d7248a5f5146dcfbd2bb44e